fastlane/lib/fastlane/actions/create_keychain.rb

```diff
@@ -51,6 +52,8 @@ def self.run(params)
 
         commands << Fastlane::Actions.sh(command, log: false)
 
+        Actions.lane_context[Actions::SharedValues::KEYCHAIN_NAME] = keychain_path.split('/').last
```

Comment:
not tied to this implementation - but did this to cater for both `:name` or `:path` options

Reply:
not sure if we want to test for the `name: 'my/key/chain'` case